docs/user/sbt.rst

```diff
@@ -43,13 +43,13 @@ Scala versions
 
 Scala Native supports following Scala versions for corresponding releases:
 
-==================== ================
+==================== ========================
 Scala Native Version Scala Versions
-==================== ================
+==================== ========================
 0.1.x                2.11.8
 0.2.x                2.11.8, 2.11.11
-0.3.x                2.11.8, 2.11.11
-==================== ================
+0.3.x                2.11.8, 2.11.11, 2.11.12
+==================== ========================
```

Comment:
This needs to be clarified, not all 0.3.x releases are going to support 2.11.12.

I'd add split in two lines:

```
0.3.0-0.3.3               2.11.8, 2.11.11
0.3.4+                    2.11.8, 2.11.11, 2.11.12
```